### DISCUSSION TITLE
Making Zoomable work on composables that are not images

### DISCUSSION THREAD

**Discussion by hahouari** [mxalbert1996/Zoomable#25] [Ideas]
A huge thanks for the time you took updating and maintaining this `Zoomable` composable, I know this project is focused on images zooming/panning, and I want to know if it is possible to make it work with composable such as `Box`.

In my case this project was almost perfect, all I needed was to wrap a `Canvas` with `Zoomable`, but zooming is not working, I tried to tweak the size of the canvas, scale it, and wrap it around a `Box`, It even took me a full day trying to debug the `Zoomable` composable itself but I've got no luck.

I must say I'm relatively new to Jetpack Compose (Slightly more than 1 month), If It would take a lot of effort to realize such feature the least help or guidance that I get from you is much welcome and appreciated.

Again, Thank you.

**Comment by mxalbert1996** [comment]
Sorry I didn't notice this.
I think `Canvas` should just work, except that if you are drawing something density-aware such as text you may need to multiply the density by `zoomableState.scale`.
Can you provide more details on what you are drawing in your canvas and what doesn't work for you, ideally with code?
